inst/include/frogger.hpp

```diff
@@ -30,7 +30,51 @@ struct Leapfrog {
     State initial_state = {};
     initial_state.reset();
     if constexpr (ModelVariant::run_demographic_projection) {
-      initial_state.dp.p_total_pop = pars.dp.base_pop;
+
+      // Initialise base year ouputs
+      const auto& p_dp = pars.dp;
+      auto& is_dp = initial_state.dp;
```

Comment:
How do you feel about using base rather than initial?

Reply:
I guess could get confusing with what is a state space and what is an input? although the prefix should still differentiate that.